src/go/rpk/pkg/cli/debug/remotebundle/start.go

```diff
@@ -155,7 +155,7 @@ status, run:
 		},
 	}
 	f := cmd.Flags()
-	f.StringVar(&jobID, "job-id", "", "ID of the job to start debug bundle in")
+	f.StringVar(&jobID, "job-id", "", "UUID of the job to start the debug bundle in")
```

Comment:
```suggestion
	f.StringVar(&jobID, "job-id", "", "Custom UUID to assign to the job that generates the debug bundle.")
```

Reply:
Changed, thanks. :+1: 

Note: I didn't add a period at the end, as we don't add them to the flags' help text.